### PR TITLE
[rapidstring] Remove port

### PR DESCRIPTION
**Pull request by PhoebeHui**
Related issue https://github.com/microsoft/vcpkg/issues/9115 and PR https://github.com/microsoft/vcpkg/pull/8912

The author of rapidstring delete their Github repro, update the repo to cenit/rapidstring.

Background:
https://github.com/microsoft/vcpkg/pull/8912#issuecomment-551969108
https://github.com/microsoft/vcpkg/pull/8912#issuecomment-558246277

rapidstring has no features need test locally.

### CODE REVIEW

File: ports/rapidstring/CONTROL [hunk 1]
```diff
@@ -1,3 +1,3 @@
 Source: rapidstring
-Version: 2018-08-03
+Version: 2019-11-29
```
Comment (NancyLi1013) [inline ports/rapidstring/CONTROL]: Please also add homepage
Homepage: https://github.com/cenit/rapidstring

**Review by NancyLi1013 — CHANGES_REQUESTED** [review]

**Review by NancyLi1013 — APPROVED** [review]

File: ports/rapidstring/CONTROL (hunk 1 above)
Comment (PhoebeHui) [reply ports/rapidstring/CONTROL]: Thanks!

**Comment by grdowns** [comment]
Thanks for doing this @PhoebeHui!

@cenit just a heads up, I've placed a message in the portfile letting the user know that you own the fork in question. If you'd prefer us to not do this, just tell us and we can revert it for you. Thanks!

**Comment by cenit** [comment]
@grdowns ok sure, just to be clear that this is not from @boyerjohn. I just hope that people are not expecting me to do the development he just stopped, apart from that this repo will not disappear :)
I might also accept interesting PR in future if they will happen

**Comment by grdowns** [comment]
After some thought, we should remove rapidstring unless we have someone willing to keep it maintained. To anyone reading, if this sounds like you then feel free to create a PR. We would be more than happy to have rapidstring again if there is interest :slightly_smiling_face:

**Comment by timautry** [comment]
:( disappointing :( I guess that's life though :) Maybe someone else whose is willing and able may pick it up in the future :)

**Pull request merged**